src/box/txn_limbo.h

```diff
@@ -502,6 +502,13 @@ txn_limbo_has_owner(struct txn_limbo *limbo)
 	return limbo->owner_id != REPLICA_ID_NIL;
 }
 
+/** Return whether limbo is owned by current instance. */
+static inline bool
+txn_limbo_is_owned_by_current_instance(struct txn_limbo *limbo)
+{
+	return limbo->owner_id == instance_id;
```

Comment:
`instance_id` is to some extent a magic value (at least because it is a global variable) , and the comparisons I saw in the code looked uncomfortable to me. I added this helper for clarity to explain what's going on.

Reply:
Ok, fair enough 💪.